### ISSUE TITLE
[Beta] Widget Crash

### ISSUE THREAD

**Issue by linuskr** [ItsCalebJones/SpaceLaunchNow-Android#196]
Hi,
The Widget keeps crashing for me with:
```
FATAL EXCEPTION: main
Process: me.calebjones.spacelaunchnow, PID: 29035
java.lang.RuntimeException: Unable to start receiver me.calebjones.spacelaunchnow.widgets.launchlist.LaunchListWidgetProvider: java.lang.IllegalArgumentException: me.calebjones.spacelaunchnow: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:4315)
	at android.app.ActivityThread.-$$Nest$mhandleReceiver(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2153)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7872)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:946)
Caused by: java.lang.IllegalArgumentException: me.calebjones.spacelaunchnow: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
	at android.app.PendingIntent.checkFlags(PendingIntent.java:401)
	at android.app.PendingIntent.getBroadcastAsUser(PendingIntent.java:671)
	at android.app.PendingIntent.getBroadcast(PendingIntent.java:658)
	at me.calebjones.spacelaunchnow.widgets.launchlist.LaunchListManager.buildWidget(SourceFile:54)
	at me.calebjones.spacelaunchnow.widgets.launchlist.LaunchListManager.updateAppWidget(SourceFile:3)
	at me.calebjones.spacelaunchnow.widgets.launchlist.LaunchListWidgetProvider.onUpdate(SourceFile:4)
	at android.appwidget.AppWidgetProvider.onReceive(AppWidgetProvider.java:66)
	at me.calebjones.spacelaunchnow.widgets.launchlist.LaunchListWidgetProvider.onReceive(SourceFile:16)
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:4306)
	... 9 more
```
I'm using the playstore beta version.

**Comment by linuskr** [comment]
Exact Version:
Version 3.15.0-b19
versionCode 4500019

**Comment by ItsCalebJones** [comment]
fixed with 172945be2c9f9972069f49b42b04f602749c6d51
